create/templates/beacon_extension/extension.config.yml

```diff
@@ -0,0 +1,12 @@
+runtime_context: sandbox
+version: 2
+configuration:
+  type: object
+  fields:
+    trackingId:
+      name: Tracking Id
```

Comment:
we should probably restrict name to be a string with no spaces, given that it's going to how they reference it in the code (different from script's use case which is a label in a UI)